### ISSUE TITLE
[PostgreSQL] Comments for tables are missing in DDL

### ISSUE THREAD

**Issue by lemon-juice** [dbeaver/dbeaver#3078]
When I edit a table with a comment the comment is shown as Description in the Properties tab but when I go to the DDL section the comment DDL is not there under the CREATE TABLE code. It should be like this:

`COMMENT ON TABLE public.tablename IS 'my table comment';`

**Comment by jerherder** [comment]
Hello

in the new version 4.3. 5, the comments are present in the ddl script

**Comment by lemon-juice** [comment]
I'm using 4.3.5 and I can't see the comments. I see this:
![dbeaver-comment](https://user-images.githubusercontent.com/11840364/36967777-f8ed4be0-2060-11e8-9b60-a39cac331d4f.png)

**Comment by jerherder** [comment]
[comment_postgres.docx](https://github.com/dbeaver/dbeaver/files/1780332/comment_postgres.docx)

yes, sorry, i see only the comment on column and not on a the table

**Comment by jerherder** [comment]
Hello

in the version 5.0.2, the DDL script doesn't contain the comment on the table

[problem_comment_table_DDL.docx](https://github.com/dbeaver/dbeaver/files/1906858/problem_comment_table_DDL.docx)

**Comment by cristreet** [comment]
Table comments and Foreign Key comments are not showed in the ER. Is that possible?

**Comment by serge-rider** [comment]
@cristreet Yes. It is not supported (yet). Please create a feature request.

**Comment by slyapustin** [comment]
Any chance this will be reopened @serge-rider @lemon-juice @cristreet @jerherder @bkyle ?
